var/spack/repos/builtin/packages/tmux/package.py

```diff
@@ -57,6 +58,8 @@ class Tmux(AutotoolsPackage):
     depends_on("automake", when="@master")
     depends_on("autoconf", when="@master")
 
+    depends_on("yacc", when="@3.4:")
```

Comment:
It seems like the `AC_PROG_YACC` was added in https://github.com/tmux/tmux/commit/bf2cf33fc633dd6f28d0d57e669cbd0b109a5f2d. Should this not be `@3:` then?

Reply:
Should probably be `depends_on("yacc", ..., type="build")`